DXRMissions/DeusEx/Classes/DXRMissionsM05.uc

```diff
@@ -222,6 +222,7 @@ function AddMissionGoals()
     local DeusExGoal newGoal;
 
     if(dxr.localURL != "05_NYC_UNATCOMJ12LAB") return;
+    if(dxr.flags.IsReducedRando()) return;
```

Comment:
Maybe the parent shouldn't be calling AddMissionGoals() at all in reduced Rando modes?